settings.py

```diff
@@ -0,0 +1,38 @@
+"""
+settings module for the app
+"""
+
+FIRMS_LIST = [111, 7821, 126396, 250204]
+
+PUBLICATION_ID = 2
+
+#
```

Comment:
why to keep that?

Reply:
PUBLICATION_ID is needed to create a n url